apps/website/types/shared/ui.ts

```diff
@@ -35,3 +35,20 @@ export enum MarkersLegendSize {
   SMALL = 'small',
   MEDIUM = 'medium'
 }
+
+export enum ToastType {
+  SUCCESS = 'success',
+  ERROR = 'error',
+  WARNING = 'warning',
+  INFO = 'info'
+}
+
+export type ToastColor = 'success' | 'error' | 'warning' | 'info';
```

Comment:
Looks good, now since ToastColor and ToastIcon seem to be internal part of toast logic, these two types should be better removed